recipes/itak/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "2.0.1" %}
+{% set name = "itak" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lab.moilab.net/download/iTAK-{{ version }}.tar.gz
+  sha256: 24179ec86b6bde2e7ab99ef94c9eeefd18ed1b73d753482f9ff30f2080d88283
+
+build:
+  number: 0
+  noarch: python  #conflict with skip: true  # [not linux or not x86_64]
```

Comment:
No sure I understand the need for `#conflict with skip: true  # [not linux or not x86_64]`

Reply:
I will remove this comment